environment.yml

```diff
@@ -71,7 +71,7 @@ dependencies:
   - lxml  # pandas.read_html
   - openpyxl  # pandas.read_excel, DataFrame.to_excel, pandas.ExcelWriter, pandas.ExcelFile
   - pyarrow>=0.9.0  # pandas.read_paquet, DataFrame.to_parquet, pandas.read_feather, DataFrame.to_feather
-  - pyqt  # pandas.read_clipbobard
+  - pyqt>=5.9.2  # pandas.read_clipbobard
```

Comment:
can you fix the spelling here

Reply:
don't think so, i tried to fix the spelling here at beginning, but since it looks like `pyqt5` is not in conda, so the building process failed @jreback